vital/CMakeLists.txt

```diff
@@ -251,6 +251,7 @@ target_link_libraries( vital
                   vital_exceptions
                   vital_logger
                   vital_util
+                  Eigen3::Eigen
```

Comment:
I thought Eigen was header only.  Is that no longer the case?  Or am I misunderstanding what this is doing?

Reply:
> I thought Eigen was header only.

It is. `Eigen3::Eigen` is an `INTERFACE` target that carries `INTERFACE_INCLUDE_DIRECTORIES` so that other targets using it will pick up the Eigen headers. Using `target_include_directories(PUBLIC)` would have a similar effect, except a) that's more verbose, and b) if a consumer needs to use Eigen from somewhere other than where KWIVER got it, will need to manually add the include directories or they won't compile.

The downside (versus `target_include_directories`) is that consumers need to know about the `Eigen3::Eigen` target, but that's addressed by the second patch.